sdk/storage/storage-file-datalake/README.md

```diff
@@ -0,0 +1,520 @@
+# Azure Storage File Data Lake client library for JavaScript
+
+Azure Data Lake Storage (ADLS) includes all the capabilities required to make it easy for developers, data scientists, and analysts to store data of any size, shape, and speed, and do all types of processing and analytics across platforms and languages. It removes the complexities of ingesting and storing all of your data while making it faster to get up and running with batch, streaming, and interactive analytics.
+
+This project provides a client library in JavaScript that makes it easy to consume Microsoft Azure Storage Data Lake service.
+
+Use the client libraries in this package to:
+  - Create/List/Delete File Systems
+  - Create/Read/List/Update/Delete Paths, Directories and Files
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file-datalake) |
+[Package (npm)](https://www.npmjs.com/package/@azure/storage-file-datalake) |
+[API Reference Documentation](https://docs.microsoft.com/javascript/api/@azure/storage-file-datalake) |
+[Product documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction?toc=%2fazure%2fstorage%2fblobs%2ftoc.json) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file-datalake/samples) |
+[Azure Storage Data Lake REST APIs](https://docs.microsoft.com/en-us/rest/api/storageservices/data-lake-storage-gen2)
+
+## Key concepts
+
+Azure Data Lake Storage Gen2 was designed to:
+
+- Service multiple petabytes of information while sustaining hundreds of gigabits of throughput
+- Allow you to easily manage massive amounts of data
+
+Key Features of DataLake Storage Gen2 include:
+
+- Hadoop compatible access
+- A super set of POSIX permissions
```

Comment:
minor: some ends with '.' some without, would better keep align

Reply:
Thanks! For short words we don't have dot ending. For long sentences will end with dot.